### DISCUSSION TITLE
Run from terraform cloud never finishes

### DISCUSSION THREAD

**Discussion by supernovae** [kube-hetzner/terraform-hcloud-kube-hetzner#1101] [Q&A]
Anyone able to get this to build from terraform cloud?

I put my ssh secrets in as encrypted variables and everything plasses tfplan and I see all of the infrastructure get created on hetzner but it seems the validation for the workers/agents never finish..

I had similar problem at home since the firewall on my ubiquity dream machine seems to block the ssh check (which is why i tried tf cloud) but figured i'd ask if anyone got it working or if it can't work because of the check type

**Comment by SamKingDev** [comment]
Did you ever resolve this?
